shopelectro/templatetags/se_extras.py

```diff
@@ -85,6 +85,13 @@ def humanize_price(price):
     return intcomma(floatformat(price, 0))
 
 
+@register.filter
+def show_price_units(item):
```

Comment:
We are showing a price. Units string is just a representation of this price.
```suggestion
def show_price_in_units(item: Product):
```

And one more point. It seems that a product itself could have to represent it's price. So, we should move it to Product class method.

It's up to you. But if you'll decide to preserve templates filter, tell plz why you are doing it. Your explanation will be good both for me and for project

Reply:
>And one more point. It seems that a product itself could have to represent it's price. So, we should move it to Product class method.

I think so too, but besides Product model we have to deal with Cart's positions and Position model, so probably sooner or later we will have to share this logic for them too
Therefore I decide to do it now